drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -310,4 +314,55 @@ protected String getPropertyHeaderTitle(ScenarioSimulationContext context, List<
                 .findFirst()
                 .map(FactMapping::getExpressionAlias);
     }
+
+    /**
+     * It checks if the provided instance title is valid. A title is valid if not already present in his own column group
+     * (GIVEN/EXPECTED). The only exception is when the selected column is part (i.e on the left or on the right) of
+     * the the same instance group of the selected column (i.e columns with the same <code>factIdentifier</code> and in
+     * the same Range).
+     * @param context
+     * @param selectedColumn It's the user selected column
+     * @param factIdentifier It's the factIdentifier to assign to selectedColumn
+     * @param title It's the title to assign to selectedColumn
+     * @return
+     *
+     */
+    protected boolean isValidInstanceTitle(ScenarioSimulationContext context,
+                                           ScenarioGridColumn selectedColumn,
+                                           FactIdentifier factIdentifier,
+                                           String title) {
+        String columnGroup = selectedColumn.getInformationHeaderMetaData().getColumnGroup();
+        String groupName = columnGroup.contains("-") ? columnGroup.substring(0, columnGroup.indexOf("-")) : columnGroup;
```

Comment:
Can you please explain the reason of this substring?

Reply:
I refactored this part as agreed, now it should be easy to handle this subColumnGroup name handling.